From a7604c770435477482bbe48c2336964fcef45b7b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20Rodr=C3=ADguez?= <crrodriguez@opensuse.org>
Date: Tue, 28 Jan 2014 16:41:08 -0300
Subject: [PATCH] skipcpio: return something at end of program

Otherwise the compiler emits a warning and the return vale is
in theory undefined.
---
 skipcpio/skipcpio.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/skipcpio/skipcpio.c b/skipcpio/skipcpio.c
index fbf391b..c912061 100644
--- a/skipcpio/skipcpio.c
+++ b/skipcpio/skipcpio.c
@@ -120,4 +120,6 @@ int main(int argc, char **argv)
                         break;
         }
         fclose(f);
+
+        return EXIT_SUCCESS;
 }
